test/server/test_image_validation.py

```diff
@@ -0,0 +1,84 @@
+from urlparse import urljoin
+from clientlib import (
+    make_example_shot,
+    make_random_id,
+    screenshots_session,
+    example_images
+)
+import random
+
+# Hack to make this predictable:
+random.seed(0)
+
+
+def image_setup():
+    for image in example_images:
+        valid_data_image = image['url']
+        return valid_data_image
+
+
+def user_setup():
+    with screenshots_session() as user:
+        shot_id = make_random_id() + "/test.com"
+        shot_data = urljoin(user.backend, "data/" + shot_id)
+        shot_json = make_example_shot(user.deviceId)
+        return (shot_data, shot_json, user)
+
+
+def test_invalid_data_image():
+    image = image_setup()
+    (shot_data, shot_json, user) = user_setup()
+    valid_header = "data:image/png;base64,iVBORw0KGgo"
+    invalid_header = "data:image/png;base64,R0k"
+    if valid_header in image:
+        invalid_data_image = image.replace(valid_header, invalid_header)
+        for clip_id in shot_json['clips']:
+            shot_json['clips'][clip_id]['image']['url'] = invalid_data_image
```

Comment:
This still ends up being too long and doesn't hit the invalid-data-image code path.

@leni1 can you set the url here to `invalid_header`? and delete the `image.replace` two lines up?